doc/configuration.rst

```diff
@@ -233,6 +233,41 @@ Currently available are:
 Used by:
   - `NetworkPowerDriver`_
 
+NetworkPowerPortWithAuth
+++++++++++++++++
+A NetworkPowerPortWithAuth describes a remotely switchable power port.
+
+.. code-block:: yaml
+
+   NetworkPowerPort:
+     model: spe_switch
```

Comment:
I think the model `spe_switch` is not precise enough. The Telnet API ist he API of the FL SWITCH 2303-8SP other manufacturer of spe switch device can have an other API.
Please rename the model

Reply:
Renamed to *phoenix_fl_switch*